src/types/futures.ts

```diff
@@ -428,6 +428,42 @@ export interface FuturesAccountBalance {
   marginAvailable: boolean;
   updateTime: numberInString;
 }
+export interface Asset {
+  asset: string;
+  walletBalance: numberInString;
+  unrealizedProfit: numberInString;
+  marginBalance: numberInString;
+  maintMargin: numberInString;
+  initialMargin: numberInString;
+  positionInitialMargin: numberInString;
+  openOrderInitialMargin: numberInString;
+  maxWithdrawAmount: numberInString;
+  crossWalletBalance: numberInString;
+  crossUnPnl: numberInString;
+  availableBalance: numberInString;
+  marginAvailable: boolean;
+  updateTime: number;
+}
+
+export interface Position {
+  symbol: string;
+  initialMargin: numberInString;
+  maintMargin: numberInString;
+  unrealizedProfit: numberInString;
+  positionInitialMargin: numberInString;
+  openOrderInitialMargin: numberInString;
+  leverage: numberInString;
+  isolated: boolean;
+  entryPrice: numberInString;
+  maxNotional: numberInString;
+  positionSide: PositionSide;
+  positionAmt: numberInString;
+  notional: numberInString;
+  isolatedWallet: numberInString;
+  updateTime: number;
+  bidNotional: numberInString;
+  askNotional: numberInString;
+}
```

Comment:
```suggestion
export interface FuturesAccountAsset {
  asset: string;
  walletBalance: numberInString;
  unrealizedProfit: numberInString;
  marginBalance: numberInString;
  maintMargin: numberInString;
  initialMargin: numberInString;
  positionInitialMargin: numberInString;
  openOrderInitialMargin: numberInString;
  maxWithdrawAmount: numberInString;
  crossWalletBalance: numberInString;
  crossUnPnl: numberInString;
  availableBalance: numberInString;
  marginAvailable: boolean;
  updateTime: number;
}

export interface FuturesAccountPosition {
  symbol: string;
  initialMargin: numberInString;
  maintMargin: numberInString;
  unrealizedProfit: numberInString;
  positionInitialMargin: numberInString;
  openOrderInitialMargin: numberInString;
  leverage: numberInString;
  isolated: boolean;
  entryPrice: numberInString;
  maxNotional: numberInString;
  positionSide: PositionSide;
  positionAmt: numberInString;
  notional: numberInString;
  isolatedWallet: numberInString;
  updateTime: number;
  bidNotional: numberInString;
  askNotional: numberInString;
}
```

Thanks for the quick PR! Tiny nitpick (since it's hard to rename these once they're out in the wild), would prefer giving a unique-ish name constraining this type to the futures (in case more similar types are added for other categories). Annoyingly, it seems it's not compatible with the existing `FuturesPosition` type, so a slightly different name like this?